inference-engine/thirdparty/clDNN/CMakeLists.txt

```diff
@@ -757,8 +757,6 @@ foreach(__CLDNN_CompilerFlagName IN ITEMS "CMAKE_CXX_FLAGS" "CMAKE_C_FLAGS")
         MultiProcessorCompilation
         DeadCodeEliminate
         ExtensionsEnabled
-        TreatWarnAsErrorEnabled
-        WarnLevel4
```

Comment:
What are the reasons of this commit? You are removing valuable options and required declarations